pkg/web/templates/partials/sunrise/message_preview.html

```diff
@@ -0,0 +1,182 @@
+      <p class="py-2">Your prompt action is needed to ensure compliance with the Travel Rule.</p>
+      <p class="py-2">Best regards,</p>
+      <p class="py-2">{originator_name}</p>
+    </div>
+  </section>
+  <div>
+    <p class="pb-4">The following information will be displayed to the counterparty after clicking the secure link. Please confirm before sending. Click cancel to edit.</p>
+    <section class="mx-auto mb-4 p-4 md:mx-0 lg:mb-0 md:p-6 border border-black rounded bg-neutral">
+      <div class="grid md:grid-cols-2">
+        <div class="mb-4">
+          <h2 class="pb-2 text-lg font-bold">TRANSACTION DETAILS</h2>
+          <dl>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Originator Crypto Address</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Amount</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Network</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Asset Type</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Transaction ID/Hash</dt>
+              <dd></dd>
+            </div>
+          </dl>
+        </div>
+        <div>
+          <h2 class="pb-2 text-lg font-bold">BENEFICIARY</h2>
+          <dl>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Beneficiary Crypto Address</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Beneficiary Email Address</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">First Name</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Last Name</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Country</dt>
+              <dd></dd>
+            </div>
+          </dl>
+        </div>
+      </div>
+      <div class="my-4 grid md:grid-cols-2">
+        <div class="mb-4">
+          <h2 class="pb-2 text-lg font-bold">ORIGINATOR</h2>
+          <dl>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Name Identifiers</dt>
+              <dd></dd>
+            </div>
+            <div class="transaction-preview-container">
+              <dt class="font-semibold">Name Identifiers Type</dt>
+              <dd class="identifier-type badge-style"></dd>
```

Comment:
The style set for the types appears in the modal as an empty blue circle. Once the endpoint is implemented, text will either appear inside like a badge or the type will not be displayed if one isn't provided.